roles/vmware-ova/ovfconf/files/ovfconf

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+ADDRESS=`/usr/bin/vmtoolsd --cmd "info-get guestinfo.ovfenv" | /bin/grep 'key="address"' | /bin/sed 's/.*value="\([^"]*\)".*/\1/'`
+GATEWAY=`/usr/bin/vmtoolsd --cmd "info-get guestinfo.ovfenv" | /bin/grep 'key="gateway"' | /bin/sed 's/.*value="\([^"]*\)".*/\1/'`
+
+if [ ! -z "${ADDRESS}" ]; then
+    /bin/sed -i "s~address dhcp~address ${ADDRESS}~" /opt/vyatta/etc/config/config.boot
+fi
+if [ ! -z "${GATEWAY}" ]; then
+    cat <<EOF >>/opt/vyatta/etc/config/config.boot
```

Comment:
We should not be using this approach. When it was used, it was for lack of better tools for the job. The new vyos.configtree library is already used in some migration scripts, you can find examples in https://github.com/vyos/vyos-1x/blob/current/src/migration-scripts/system/7-to-8 and https://github.com/vyos/vyos-1x/blob/current/src/migration-scripts/config-management/0-to-1